apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -80,6 +81,21 @@ export class PortfolioService {
     private readonly symbolProfileService: SymbolProfileService
   ) {}
 
+  public getAnnualizedPerformancePercent({
+    daysInMarket,
+    netPerformancePercent
+  }: {
+    daysInMarket: number;
+    netPerformancePercent: number;
+  }) {
+    if (isNumber(daysInMarket) && daysInMarket > 0) {
+      const exponent = new Big(365).div(daysInMarket).toNumber();
+      return Math.pow(1 + netPerformancePercent, exponent) - 1;
```

Comment:
Why not with big?

Reply:
`pow` only works with `int` in `Big.js` → https://mikemcl.github.io/big.js/#pow

Reply:
Sad